### PR TITLE
Add theme support for charts

### PR DESCRIPTION
**Pull request by erangatl**
## Purpose
Add two themes, light and dark for charts. Sample docs can be switchable to a light and dark type themes.
Fixes: #27

## Goals
Providing light and dark theme support

## Approach
Two theme files added and 'light' type theme is set as default. The possibility of changing the theme to dark type is given through a prop.

## User stories
https://docs.google.com/document/d/1pzBdp1AVLwn0ByCb58pifztBCteeyBFywyqx6Usks9w/edit?usp=sharing

## Release note
This will provide light and dark theme support

## Documentation
Documentation added to the getting started markdown 

## Security checks
 - Followed secure coding standards in http://wso2.com/technical-reports/wso2-secure-engineering-guidelines? yes
 - Ran FindSecurityBugs plugin and verified report? yes/no
 - Confirmed that this PR doesn't commit any keys, passwords, tokens, usernames, or other secrets? yes

### CODE REVIEW

File: samples/components/Header.jsx [hunk 1]
```diff
@@ -0,0 +1,61 @@
...
+import BackIcon from 'material-ui-icons/ArrowBack';
+import GitHub from '../components/GitHub';
+import { Link } from 'react-router-dom';
+import PropTypes from 'prop-types';
+
+const style = {
+    background: '#2196f3',
+};
+
+export default class header extends React.Component {
```
Comment (this) [inline samples/components/Header.jsx]: Class name should start with capital `Header`

Comment (erangatl) [reply samples/components/Header.jsx]: Fixed

File: src/components/LineChart.jsx [hunk 2]
```diff
@@ -79,78 +80,87 @@ export default class LineChart extends BaseChart {
...
+                }]}
+            />),
         ];
     }
 
     render() {
-        const { config, height, width } = this.props;
+        const { config, height, width, yDomain, theme } = this.props;
         const { chartArray, dataSets, xScale, ignoreArray } = this.state;
+        const currentTheme = theme === 'light' ? lightTheme : darkTheme;
```
Comment (ramindu90) [inline src/components/LineChart.jsx]: @erangatl shall we remove these unnecessary spaces?

File: dist/components/resources/css/tableChart.css [hunk 3]
```diff
@@ -74,7 +74,8 @@
 }
 
 .ReactTable .rt-thead.-header {
-	box-shadow: 0 2px 15px 0 rgba(0, 0, 0, 0.15)
+	box-shadow: 0 2px 15px 0 rgba(0, 0, 0, 0.15);
```
Comment (lasanthaS) [inline dist/components/resources/css/tableChart.css]: Wrong indentiation

File: dist/components/resources/css/tableChart.css [hunk 4]
```diff
@@ -167,7 +168,7 @@
 }
 
 .ReactTable .rt-tbody .rt-tr-group {
-	border-bottom: 1px solid rgba(0, 0, 0, 0.02);
+	border-bottom: 1px solid rgba(0, 0, 0, 0.05);
```
Comment (lasanthaS) [inline dist/components/resources/css/tableChart.css]: Wrong indentation

Comment (erangatl) [reply dist/components/resources/css/tableChart.css]: Fixed with https://github.com/wso2/react-vizgrammar/pull/208

File: dist/components/resources/css/tableChart.css (hunk 3 above)
Comment (erangatl) [reply dist/components/resources/css/tableChart.css]: Fixed with https://github.com/wso2/react-vizgrammar/pull/208

**Pull request merged**